Dockerfile

```diff
@@ -27,7 +27,7 @@ ADD LICENSE /usr/share/doc/python-crane/
 ADD COPYRIGHT /usr/share/doc/python-crane/
 ADD README.rst /usr/share/doc/python-crane/
 
-RUN pip-python install /usr/local/src/crane/
+RUN pip install /usr/local/src/crane/
```

Comment:
Isn't it supposed to be pip-python? When pip is installed via rpm in [L#13](https://github.com/elyezer/crane/blob/patch-2/Dockerfile#L13) I don't think it will deliver `pip` instead it will deliver `pip-python`.

@elyezer did you experience an issue that this change resolves?
@elyezer Was this tested?